zebra/zebra_l2.c

```diff
@@ -433,6 +433,8 @@ void zebra_l2if_update_bridge_slave(struct interface *ifp,
 		if (ZEBRA_BRIDGE_MASTER_MAC_CHANGE)
 			zebra_vxlan_if_update(ifp,
 					      ZEBRA_VXLIF_MASTER_MAC_CHANGE);
+		if (ZEBRA_BRIDGE_MASTER_UP)
+			zebra_vxlan_if_update(ifp, ZEBRA_VXLIF_MASTER_CHANGE);
```

Comment:
Do you need to add any code in zebra_vxlan_if_update() for this new flag ?

Reply:
Hi Sri Mohan,
no, actually, when svi interface is down, we previously were calling that function, but `if_is_operative(zl3vni->svi_if)` returned false, and prevented from establishing evpn context.
so I reuse the same event MASTER_CHANGE, but this time, svi interface will be operative.